### ISSUE TITLE
java.lang.NoSuchMethodError: net.sf.freecol.common.util.StringUtils.join

### ISSUE THREAD

**Issue by Beep6581** [FreeCol/freecol#40]
The latest `master` no longer runs in Sabayon.

`nightly-2018-08-22-371-g`937b8cc22

```
 $ cat ~/.cache/freecol/FreeCol.log
FreeCol game version: 0.11.6 (Revision: 937b8cc22)
FreeCol protocol version: 0.1.6

Java vendor: Oracle Corporation
Java version: 1.8.0_191
Java WM name: OpenJDK 64-Bit Server VM
Java WM vendor: Oracle Corporation
Java WM version: 25.191-b12

OS name: Linux
OS architecture: amd64
OS version: 4.20.0-sabayon

java.util.logging.LogManager$RootLogger log
        WARNING: Uncaught exception from thread: Thread[main,5,main]
        Tue Apr 16 00:26:55 CEST 2019
        Thread: 1
        Stack trace:
java.lang.NoSuchMethodError: net.sf.freecol.common.util.StringUtils.join(Ljava/lang/String;Ljava/util/List;)Ljava/lang/String;
        at net.sf.freecol.common.io.FreeColDirectories.sanitize(FreeColDirectories.java:604)
        at net.sf.freecol.common.io.FreeColDirectories.getOptionsFile(FreeColDirectories.java:1062)
        at net.sf.freecol.common.io.FreeColDirectories.getClientOptionsFile(FreeColDirectories.java:743)
        at net.sf.freecol.client.ClientOptions.getSpecialOptions(ClientOptions.java:844)
        at net.sf.freecol.FreeCol.main(FreeCol.java:338)
----------------------------
```

**Comment by Beep6581** [comment]
Commit ac79247dc caused the problem.

**Comment by Beep6581** [comment]
Fixed by commit 0ad1130ac
